akka-docs/rst/java/mailboxes.rst

```diff
@@ -171,6 +175,30 @@ Akka comes shipped with a number of mailbox implementations:
 
   - Configuration name: "akka.dispatch.BoundedPriorityMailbox"
 
+* UnboundedStablePriorityMailbox
```

Comment:
@dimbleby can you also update the doc code samples to use this mailbox instead of the old one:

http://doc.akka.io/docs/akka/2.3.9/scala/mailboxes.html#Mailbox_configuration_examples
http://doc.akka.io/docs/akka/2.3.9/java/mailboxes.html#Mailbox_configuration_examples

thanks